### DISCUSSION TITLE
Rate limits for Quests

### DISCUSSION THREAD

**Discussion by ostec-benni** [browniebroke/deezer-python#432] [Q&A] [answered]
Hello, 

Is there any rate limits for requests? 

For example if I want to fetch 500 Tracks to get the `isrc` and I send 500 single requests simultaneously or sequentially. 

Thank you for your help.

**Comment by browniebroke** [comment, marked as answer]
This client library doesn't do any rate limiting, and I have no idea what happens on the server, but most likely, yes. 

Looking at the [potential API errors](https://developers.deezer.com/api/errors) that are returned, I see `QUOTA` and `ITEMS_LIMIT_EXCEEDED` which seem like they might be relevant.

PS: I'm not working at Deezer, so I have no internal knowledge here 😄
